### ISSUE TITLE
autorestic cron "--lean" flag does not work properly

### ISSUE THREAD

**Issue by olofvndrhr** [cupcakearmy/autorestic#178]
**Describe the bug**
When you use the "--lean" flan in the "cron" subcommand, the output is not silent.
"Using config:" is still displayed. With older versions this was not the case. 

**Expected behavior**
From the docs:
> "The --lean flag will omit output like skipping location x: not due yet. This can be useful if you are dumping the output of the cron job to a log file and don't want to be overwhelmed by the output log."

**Environment**
 - OS: Linux, amd64
 - Version: autorestic version 1.5.8

**Additional context**
**Example:**
```bash
[root@rs1 /opt/autorestic]# autorestic --version
autorestic version 1.5.8
[root@rs1 /opt/autorestic]# autorestic -c test.yml cron
Using config:    /opt/autorestic/test.yml
Using lock:      .autorestic.lock.yml
Skipping "test", not due yet.
[root@rs1 /opt/autorestic]# autorestic -c test.yml cron --lean
Using config:    /opt/autorestic/test.yml
```

**Code:**
In the file: "[autorestic/internal/config.go](https://github.com/cupcakearmy/autorestic/blob/master/internal/config.go)" there is an "if" statement that controls the behavior. I dont know "go", but i didnt see a mistake at a first look.
```
if !flags.CRON_LEAN {
            colors.Faint.Println("Using config: \t", absConfig)
            }
```

**Comment by bin101** [comment]
@cupcakearmy  I don't think this has been fixed:

```
root@acrux:~# autorestic --version
autorestic version 1.6.2
root@acrux:~# autorestic cron --lean
Using config: 	 /root/.autorestic.yml
```

**Comment by cupcakearmy** [comment]
Yup, was a mistake on my side, will fix in 1.7.1
